@ShevchenkoRostyslav 
Got compilation error. Could you please check that?
Thanks!

> > Building edm plugin tmp/slc6_amd64_gcc530/src/Analysis/Diagnosis/plugins/AnalysisDiagnosisAuto/libAnalysisDiagnosisAuto.so
> > tmp/slc6_amd64_gcc530/src/Analysis/BackgroundModel/src/AnalysisBackgroundModel/RooDoubleGausExp.o: In function `analysis::backgroundmodel::RooDoubleGausExp::IsA() const':
> > RooDoubleGausExp.cc:(.text._ZNK8analysis15backgroundmodel16RooDoubleGausExp3IsAEv[_ZNK8analysis15backgroundmodel16RooDoubleGausExp3IsAEv]+0x1): undefined reference to`analysis::backgroundmodel::RooDoubleGausExp::Class()'
> > tmp/slc6_amd64_gcc530/src/Analysis/BackgroundModel/src/AnalysisBackgroundModel/RooDoubleGausExp.o: In function `analysis::backgroundmodel::RooDoubleGausExp::ShowMembers(TMemberInspector&) const':
> > RooDoubleGausExp.cc:(.text._ZNK8analysis15backgroundmodel16RooDoubleGausExp11ShowMembersER16TMemberInspector[_ZNK8analysis15backgroundmodel16RooDoubleGausExp11ShowMembersER16TMemberInspector]+0xd): undefined reference to`analysis::backgroundmodel::RooDoubleGausExp::Class()'
> > tmp/slc6_amd64_gcc530/src/Analysis/BackgroundModel/src/AnalysisBackgroundModel/RooDoubleGausExp.o: In function `_GLOBAL__sub_I_RooDoubleGausExp.cc':
> > RooDoubleGausExp.cc:(.text.startup+0x3e): undefined reference to`ROOT::GenerateInitInstance(analysis::backgroundmodel::RooDoubleGausExp const_)'
> > tmp/slc6_amd64_gcc530/src/Analysis/BackgroundModel/src/AnalysisBackgroundModel/RooDoubleGausExp.o:(.data.rel.ro._ZTVN8analysis15backgroundmodel16RooDoubleGausExpE[_ZTVN8analysis15backgroundmodel16RooDoubleGausExpE]+0x1d8): undefined reference to `analysis::backgroundmodel::RooDoubleGausExp::Streamer(TBuffer&)'
> > tmp/slc6_amd64_gcc530/src/Analysis/BackgroundModel/src/AnalysisBackgroundModel/RooDoubleGausExp.o:(.data.rel.ro._ZTVN8analysis15backgroundmodel16RooDoubleGausExpE[_ZTVN8analysis15backgroundmodel16RooDoubleGausExpE]+0x650): undefined reference to`non-virtual thunk to analysis::backgroundmodel::RooDoubleGausExp::Streamer(TBuffer&)'
> > collect2: error: ld returned 1 exit status
> > config/SCRAM/GMake/Makefile.rules:1917: recipe for target 'tmp/slc6_amd64_gcc530/src/Analysis/BackgroundModel/src/AnalysisBackgroundModel/libAnalysisBackgroundModel.so' failed
> > gmake: *_\* [tmp/slc6_amd64_gcc530/src/Analysis/BackgroundModel/src/AnalysisBackgroundModel/libAnalysisBackgroundModel.so] Error 1
> > gmake: **\* Waiting for unfinished jobs....
> > Leaving library rule at src/Analysis/Diagnosis/plugins
> > gmake: **\* [There are compilation/build errors. Please see the detail log above.] Error 2

--- a/Analysis/BackgroundModel/src/classes.h
+++ b/Analysis/BackgroundModel/src/classes.h
@@ -8,3 +8,4 @@
 #include "Analysis/BackgroundModel/interface/RooQuadGausExp.h"
 #include "Analysis/BackgroundModel/interface/RooMyNovosibirsk.h"
 #include "Analysis/BackgroundModel/interface/RooExtendNovosibirsk.h"
+#include "Analysis/BackgroundModel/interface/RooDoubleGausExp.h"

--- a/Analysis/BackgroundModel/src/classes_def.xml
+++ b/Analysis/BackgroundModel/src/classes_def.xml
@@ -9,4 +9,5 @@
   <class name="analysis::backgroundmodel::RooQuadGausExp" />
   <class name="analysis::backgroundmodel::RooMyNovosibirsk" />
   <class name="analysis::backgroundmodel::RooExtendNovosibirsk" />
+  <class name="analysis::backgroundmodel::RooDoubleGausExp" />
 </lcgdict>

